### PR TITLE
vars-schema: Add HOST_CONFIG HDMI config definitions for the RPi family

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: minor
See: https://www.flowdock.com/app/rulemotion/balenalabs/threads/Zp0NS-1iY28vO0lJNih-lW4U_Ag
See: https://www.raspberrypi.org/documentation/configuration/config-txt/video.md
Signed-off-by: Thodoris Greasidis <thodoris@balena.io>

### CODE REVIEW

File: src/features/vars-schema/env-vars.ts [hunk 1]
```diff
@@ -141,6 +141,34 @@ export const DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES: Array<{
...
+			},
+			BALENA_HOST_CONFIG_hdmi_mode: {
+				type: 'string',
+				description: 'Define the HDMI output format',
+				examples: [87],
+			},
+			BALENA_HOST_CONFIG_display_rotate: {
+				type: 'string',
+				description: 'Define the rotation or flip of the display',
+				examples: ['0x10000'],
```
Comment (klutchell) [inline src/features/vars-schema/env-vars.ts:169]:
```suggestion
				examples: ['1'],
```
Rotating 90 degrees (`1`) is far more common than having to flip the image horizontally (`0x10000`).

Comment (thgreasi) [reply src/features/vars-schema/env-vars.ts:169]: let me add both actually

File: src/features/vars-schema/env-vars.ts [hunk 2]
```diff
@@ -141,6 +141,34 @@ export const DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES: Array<{
...
+				default: 0,
+			},
+			BALENA_HOST_CONFIG_hdmi_group: {
+				type: 'integer',
+				description: 'Define the HDMI output group',
+				examples: [2],
+				default: 0,
+			},
+			BALENA_HOST_CONFIG_hdmi_mode: {
+				type: 'string',
```
Comment (klutchell) [inline src/features/vars-schema/env-vars.ts:162]: This could probably be `integer` unless you saw a possible value that I missed?

Comment (thgreasi) [reply src/features/vars-schema/env-vars.ts:162]: Ooops you are right :sweat: thanks for spotting...
Is there a default value for this @klutchell ?

Comment (klutchell) [reply src/features/vars-schema/env-vars.ts]: `hdmi_mode=1` seems to be recommended to start and verify the display works, then increase it from there.

This field is ignored if `hdmi_group=0` so it shouldn't matter much what the default is here.

**Review by klutchell — DISMISSED** [review]
LGTM

**Comment by thgreasi** [comment]
@balena-ci rebase

**Comment by thgreasi** [comment]
@balena-ci I self-certify!

**Pull request merged**